### PR TITLE
Update Gemfile.lock in the 2.0 branch to use last released plugins

### PR DESCRIPTION
**Pull request by purbon**
Fixes broken http://build-eu-00.elastic.co/view/LS%202.0/job/Logstash_Default_Plugins_20/247/

### CODE REVIEW

File: Gemfile.jruby-1.9.lock [hunk 1]
```diff
@@ -724,4 +724,4 @@ DEPENDENCIES
   rspec (~> 3.1.0)
   rubyzip (~> 1.1.7)
   simplecov
-  stud (~> 0.0.19)
+  stud (~> 0.0.21)
```
Comment (jsvd) [inline Gemfile.jruby-1.9.lock]: we should force this to be ~> 0.0.22, that's where the new stud stop stuff was added

Comment (purbon) [inline Gemfile.jruby-1.9.lock]: makes sense, good point.

**Comment by jsvd** [comment]
LGTM

**Comment by purbon** [comment]
Still not merge it because looks like logstash-output-statsd is failing under the way we run the defaults plugins test.

**Pull request closed without merging**